### DISCUSSION TITLE
Can I build an application to generate images using Flowise?

### DISCUSSION THREAD

**Discussion by growmuye** [FlowiseAI/Flowise#1339] [Q&A]
Can I build an application to generate images using Flowise?

**Comment by HenryHengZJ** [comment]
we have a template example: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/9512ffe4-7f18-4a2d-9ce1-a76e275c74c8)

**Reply by growmuye** [reply]
Thank you!

**Reply by agnosus** [reply]
Hi guys, I have problem in displaying images in the deployed chatbot. In flowise console chatbot it works fine. see examples below.
do you know how to solve this problem?
thanks ,

![image](https://github.com/FlowiseAI/Flowise/assets/64579856/718ff8a8-67d7-45ef-8350-2a8076c578bb)

![image](https://github.com/FlowiseAI/Flowise/assets/64579856/f893d14b-da69-46f0-a8af-4c496b2a2ba4)

**Reply by agnosus** [reply]
@HenryHengZJ  can you advise if there is a solution?

**Reply by saisrinu135** [reply]
Hey, How are you sending images in response? Can I see your flow?

**Reply by Panconjugo23** [reply]
I'm also interested on that topic, looking for a way to retrieve images from a vector store, but not sure what nodes to use.

**Reply by saisrinu135** [reply]
Hey! I got the images in chat. I have stored the links in a CSV file and embedded that file. Just give a prompt to return the images in markdown format to see images in chat. I have also tried using Huggingface to embed images using clip but I couldn't. 
Anyone who had tried using huggingface, help me.

**Comment by ghost** [comment]
Related to this, can we not use dall-e model from Azure in Flowise?
